packages/system-tests/scenarios/forceSoqlQuery.test.ts

```diff
@@ -27,7 +27,7 @@ describe('force:soql:query commands', () => {
   let app: SpectronApplication;
   let common: CommonActions;
   // tslint:disable-next-line:no-invalid-this
-  this.timeout(10000);
+  // this.timeout(10000);
```

Comment:
@Sweetman I had to comment this out as it threw an error. Not sure why b/c it's working in your scaffolding.test.ts.

Reply:
That's fine, that line is no longer in the latest version of the scaffolding.test.ts